README.md

```diff
@@ -32,27 +32,31 @@ pip install -r test-requirements.txt
 
 Run tests
 ```
-py.test tests/
+APP_SETTINGS=TestingConfig py.test tests/
 ```
 
 Run tests with coverage
 ```
-py.test tests/ --cov=frontstage_api
+APP_SETTINGS=TestingConfig py.test tests/ --cov=frontstage_api
 ```
 
 ## Configuration
```

Comment:
Let's try and remove all of this configuration docs going forward and just point to a single place of truth

Reply:
We can do a separate PR for refactoring the configuration. This pattern is used in a handful of different services so we could have a technical backlog task to improve these configs + make them more consistent?